tidy3d/plugins/design/method.py

```diff
@@ -23,150 +21,87 @@ class Method(Tidy3dBaseModel, ABC):
     name: str = pd.Field(None, title="Name", description="Optional name for the sweep method.")
 
     @abstractmethod
-    def run(self, parameters: Tuple[ParameterType, ...], fn: Callable) -> Tuple[Any]:
+    def run(self, parameters: Tuple[ParameterType, ...], run_fn: Callable) -> Tuple[Any]:
         """Defines the search algorithm (sequential)."""
 
-    @abstractmethod
-    def sample(self, parameters: Tuple[ParameterType, ...], **kwargs) -> Dict[str, Any]:
-        """Defines how the design parameters are sampled."""
+    def _force_int(self, next_point: dict, parameters: list):
```

Comment:
return type `-> int`?

Reply:
In a way yes but it's updating a dict in-place so I didn't think it warranted a return. What's the standard for anything in-place?

Reply:
oh I see. it would be `-> None:` then.